assets/src/blocks/Gallery/GalleryFrontend.js

```diff
@@ -5,12 +5,19 @@ import { getGalleryLayout, GALLERY_BLOCK_CLASSES } from './getGalleryLayout';
 import { Lightbox } from '../../components/Lightbox/Lightbox';
 import { useLightbox } from '../../components/Lightbox/useLightbox';
 
+const getTitle = (image) => {
+  const caption = image.caption || '';
+  const credits = image.credits && ! caption.includes(image.credits)
```

Comment:
I find it a bit strange to have a space between `!` and the rest, but I think that's a PHP habit? 😁 Probably doesn't matter

Reply:
Ah right, not even a habit but a rule in our codebase at the moment (and one day, I'll write this ADR)